src/utils/degreeConversion.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Given an angle between [-360, 360], place it in range [0, 360]
+ */
+export const convertToPositiveAngle = (degree: number): number => {
```

Comment:
*style*: place in separate files, these two functions do not depend on each other